src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -143,6 +143,15 @@ private function addAccessControlSection(ArrayNodeDefinition $rootNode)
                             ->integerNode('port')->defaultNull()->end()
                             ->arrayNode('ips')
                                 ->beforeNormalization()->ifString()->then(function ($v) { return [$v]; })->end()
+                                ->beforeNormalization()->always()->then(function ($v) {
+                                    foreach ($v as $ip) {
+                                        if (false === $this->isValidIp($ip)) {
+                                            throw new \LogicException(sprintf('The given "%s" value in the "access_control" config option is not a valid IP address.', $ip));
+                                        }
+                                    }
+
+                                    return $v;
+                                })->end()
```

Comment:
Can we use the Config' validation methods? I mean `->ifTrue(function ($ips) { ... })->thenInvalid(...)` (I believe it allows for better hints about the path of the invalid node)

Reply:
I've made this change ... but I think the new message is less precise, because we now can't tell which exact IP is wrong 🤔

Reply:
You're right. I wanted to revert the change but it seems I can't push to your fork :/ Sorry for the bad suggestion!

Reply:
No worries! Reverted.

Reply:
Did you forget to push? :)

Reply:
Yes! I forgot :( Sorry. Pushed. Thanks.